apps/home/templates/apps_home/home_page.html

```diff
@@ -28,29 +27,6 @@ <h1 class="title">{{ page.page_title }}</h1>
             </div>
         </div>
     </div>
-
-
-<!-- Desktop  -->
-    {% if header_image %}
-    <div class="hero__parallax hero__desktop" style="background-image: url('{{ header_image.url }}')">
-        <div class="herounit-landingpage herounit-landingpage--default">
-            <div class="image">
-                <div class="image__image mb-0">
-                    <img class="hero__image-back" src="{{ header_image.url }}" alt="{{ header_image.alt }}" loading="lazy">
-                </div>
-                {% if page.header_image.copyright %}
-                <div class="image__copyright ds-hero-image__copyright">
-                    <p>{{ page.header_image.copyright }}</p>
-                </div>
-                {% endif %}
-
-                <div class="image__overlay ds-hero-image__overlay">
-                    <h1 class="title">{{ page.page_title }}</h1>
-                    <p>{{ page.page_subtitle }}</p>
-                </div>
-            </div>
-        </div>
-    </div>
```

Comment:
@hom3mad3 would like a second opinion on this, do you think it is better to have the above structure so display different HTML depending on desktop or mobile or to only implement the parlax only on large screens?